pkg/export/export.go

```diff
@@ -310,7 +347,7 @@ func newMetricClient(ctx context.Context, opts ExporterOpts) (*monitoring.Metric
 }
 
 // New returns a new Cloud Monitoring Exporter.
-func New(ctx context.Context, logger log.Logger, reg prometheus.Registerer, opts ExporterOpts) (*Exporter, error) {
+func New(ctx context.Context, logger log.Logger, reg prometheus.Registerer, opts ExporterOpts, lease Lease) (*Exporter, error) {
```

Comment:
> @pintohutch [`7fe3d0b` (#974)](https://github.com/GoogleCloudPlatform/prometheus-engine/pull/974/commits/7fe3d0b10a0a0258d55440ef27bc473102e02fe1#r1661116963)
> question: How will callers of the export library refresh ExporterOpts?
>
> This change looks to be focused on refreshing the Exporter only with a given ExporterOpts struct.

I will be adding this change to `ApplyConfig`. It will take an optional `ExporterOpts`. If `nil`, nothing changes: https://github.com/GoogleCloudPlatform/prometheus-engine/pull/974